src/library_noderawfs.js

```diff
@@ -74,6 +74,8 @@ mergeInto(LibraryManager.library, {
         position += stream.position;
       } else if (whence === 2) {  // SEEK_END.
         position += fs.fstatSync(stream.nfd).size;
+      } else if (whence !== 0) {  // SEEK_SET.
+        throw new FS.ErrnoError(ERRNO_CODES.EINVAL);
```

Comment:
I think the test failures are beceause of this. Should be replaced with `{{{ cDefine('EINVAL') }}}`.

Reply:
Hmm, that was a copy/paste from a few lines below, and changing that doesn't seem to fix the test.

Reply:
Oh ok, we can leave it as is then, the whole file needs to be updated.